Update readme (#18)
iiasa/emissions_harmonization_historical: README.md

([instructions here](https://pixi.sh/latest),
we found that using the pixi provided script was best on a Mac).

To create the virtual environment, navigate to the root folder of this repository and run from your shell:

```sh
pixi install
pixi run pre-commit install
```

> This will create a python environment in "{LOCALPATH}\emissions_harmonization_historical\.pixi\envs\default\", with "python.EXE" in that folder being the default python interpreter (to be used if your for instance selecting a kernel for running a jupyter notebook in VSCode or any other IDE).

Reviewer: ```suggestion

This will create a python environment in "{LOCALPATH}/emissions_harmonization_historical/.pixi/envs/default/", with a Python executable in that folder. You should set this Python interpreter as the default in your IDE (to be used if your for instance selecting a kernel for running a jupyter notebook in VSCode or any other IDE).
```

It's a bit weird to have this environment management stuff in the README (you'd have to duplicate it in every repo you worked with, it would be like including a guide to git in the README), but if you really want it, keep it (just please don't use windows paths, they look so broken).

Author: Yes fair, the copying of a windows path was my oversight.

pixi run jupyter lab
```

> You will likely be running the notebooks in the notebook folder, and may therefore want to create .ipynb versions of the .py scripts. To do this, you can run `jupytext` (see [jupytext docs](https://jupytext.readthedocs.io/en/latest/using-cli.html)) in the environment created with pixi, doing something like:
> ```sh
> pixi run jupytext --to notebook .\notebooks\0101_CEDS-prepare.py
> pixi run jupytext --to notebook .\notebooks\0102_GFED4-prepare.py
> pixi run jupytext --to notebook .\notebooks\0103_GFED4-BB4CMIP-prepare.py
> ```

Reviewer: ```suggestion
```

I'd recommend not doing this as it scales terribly and you shouldn't need it. Just right-click on the `.py` file and select "open as jupytext" (see attached screenshot).

<img width="1439" alt="Screenshot 2024-10-22 at 5 20 03 pm" src="https://github.com/user-attachments/assets/f4bc5326-5402-4b43-9a8b-1753245228b8">

Author: Addressed in [b2b0209](https://github.com/iiasa/emissions_harmonization_historical/pull/18/commits/b2b0209639c59efb6e62fb2f3b0194f9c5bbbe77)
